src/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/MarshalTests.cs

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Xunit;
+
+namespace System.Runtime.InteropServices
+{
+    public static class MarshalTests
+    {
+        [Fact]
+        public static void AllocHGlobal_Int32_Success()
+        {
+            IntPtr p = Marshal.AllocHGlobal(100);
+            Assert.NotEqual(IntPtr.Zero, p);
```

Comment:
I do that in the test below; didn't seem like it would need to be done in every test.  Here I just wanted to make sure that the various overloads of AllocHGlobal produced a non-zero IntPtr.